### PR TITLE
Update to RC-27

### PR DESCRIPTION
**Pull request by md5sha256**
## Short Description

Might have messed up the format of a few imports but otherwise the title is explanatory. 

I've yet to actual run/test the changes in game* I'll update this comment when I or someone else can validate this doesn't break anything

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with base Slimefun and made sure nothing breaks/unexpected happens.
- [ ] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added which may not be obvious to maintainers.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

File: src/main/java/dev/j3fftw/litexpansion/Items.java [hunk 1]
```diff
@@ -21,11 +23,11 @@
...
-    public static final Category LITEXPANSION = new Category(
-        new NamespacedKey(LiteXpansion.getInstance(), "litexpansion"),
-        new CustomItem(SkullItem.fromHash("3f87fc5cbb233743a82fb0fa51fe739487f29bcc01c9026621ecefad197f4fb1"),
-            "&7LiteXpansion")
-    );
+    public static final ItemGroup LITEXPANSION = new ItemGroup(
+            new NamespacedKey(LiteXpansion.getInstance(), "litexpansion"),
+            new CustomItemStack(PlayerHead.getItemStack(PlayerSkin.fromHashCode("3f87fc5cbb233743a82fb0fa51fe739487f29bcc01c9026621ecefad197f4fb1")),
+                    "&7LiteXpansion"));
+
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/Items.java]: formatting

**Review by J3fftw1 — CHANGES_REQUESTED** [review]
besides import formatting looks good
and the small formatting

**Comment by J3fftw1** [comment]
also compile error?

File: src/main/java/dev/j3fftw/litexpansion/Items.java (hunk 1 above)
Comment (md5sha256) [reply src/main/java/dev/j3fftw/litexpansion/Items.java]: whats wrong here?

**Review by J3fftw1 — CHANGES_REQUESTED** [review]
lgtm

**Pull request merged**